app/routes.py

```
@@ -749,35 +917,44 @@ async def meetingbaas_webhook(request: Request):
                 for internal_id, details in MEETING_DETAILS.items():
                     if len(details) > 2 and details[2] == bot_id:
-                        potential_file = os.path.join(transcript_dir, f"{internal_id}.json")
+                        potential_file = os.path.join(
+                            transcript_dir, f"{internal_id}.json"
+                        )
                         if os.path.exists(potential_file):
                             transcript_file = potential_file
                             break
 
                 # If still not found, try to find any recent transcript
                 if not transcript_file and os.path.exists(transcript_dir):
```

Ruff (0.15.20)

[warning] Async functions should not use os.path methods, use trio.Path or anyio.path

(ASYNC240)